packages/kbn-legacy-logging/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "target",
+    "stripInternal": false,
+    "declaration": true,
+    "declarationMap": true,
+    "types": ["jest", "node"]
+  },
+  "include": ["./src/**/*.ts", "./typings/**/*.d.ts"]
```

Comment:
`typings ` can be removed as soon as https://github.com/elastic/kibana/pull/78143 merged